packages/components/src/Button/Button.tsx

```diff
@@ -22,6 +22,7 @@ type ButtonType =
   | 'tertiary'
   | 'positive'
   | 'negative'
+  | 'secondary_negative'
```

Comment:
All good for me, only 1 thing, change this to camelCase? We use camelCase also with colors in `packages/components/src/colors.ts`

Reply:
yep nice one

Reply:
Please also add a change log entry about this to `releases.stories.mdx`. 

```
<ReleasePill date="May 2023" version="1.3.0" />

- ✨ Add a new button type `secondaryNegative` to [Button](?path=/docs/controls-button--button)
```